src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getArea(Figure figure);
```

Comment:
Pay attention - you need to remove the argument. 
![image](https://user-images.githubusercontent.com/20326858/204789243-c03145b2-7313-4f40-baca-bf4102b0b886.png)